packages/amplify-graphql-model-transformer/src/resolvers/dynamodb/mutation.ts

```diff
@@ -234,9 +234,11 @@ export const generateCreateInitSlotTemplate = (modelConfig: ModelDirectiveConfig
     ),
   ];
 
+  if (initializeIdField) {
+    statements.push(qref(methodCall(ref('ctx.stash.defaultValues.put'), str('id'), methodCall(ref('util.autoId')))));
```

Comment:
Can we extract the qref expression into a standalone utility & reuse it everywhere we set an autoId?